shoop/xtheme/plugins/widgets.py

```diff
@@ -90,3 +91,38 @@ def decompress(self, value):
         else:
             out_dict[FALLBACK_LANGUAGE_CODE] = value
         return out_dict
+
+
+class XThemeModelChoiceWidget(forms.Select):
+    def render(self, name, value, attrs=None, choices=()):
+        return mark_safe(
+            render_to_string("shoop/xtheme/_model_widget.jinja", {
+                "name": name,
+                "selected_object": [object for pk, object in self.choices if pk == int(value)][0] if value else None,
+                "objects": self.choices,
+            })
+        )
+
+
+class XThemeModelChoiceField(forms.ModelChoiceField):
+    def __init__(self, queryset, empty_label="---------", required=True, label=None, initial=None, *args, **kwargs):
```

Comment:
Would probably be better to just `*args, **kwargs` here (or really just `**kwargs` should be enough given the use).